src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -137,6 +137,12 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
             brickRoadIndicator: hasMembersError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
             routeName: SCREENS.WORKSPACE.MEMBERS,
         },
+        {
+            translationKey: 'workspace.common.categories',
+            icon: Expensicons.Folder,
+            action: singleExecution(waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES.getRoute(policyID)))),
+            routeName: SCREENS.WORKSPACE.CATEGORIES,
+        },
```

Comment:
Noting here something that needs to be updated:
- The Categories page will be turned on by default for new collect policies in More Features page
- Thats why we are adding it before theMore features page is implemented
- This needs to be **hidden/ not accessible** for free policies
- Only available for paid group policies

Reply:
@mountiny Since the more features page has not been implemented yet, should I go with showing this functionality only for beta users and ensure it's available only for paid group policies, as mentioned?

Reply:
@ArekChr I think we can handle it in this issue https://github.com/Expensify/App/issues/36606 where the behaviour has been described I think

Reply:
In addition to only be shown for collect policies, we only want to show this to admins too, right?

Reply:
Correct, only admins can see and manage categories.